shibboleth/resources/prod/shibboleth2.xml

```diff
@@ -0,0 +1,160 @@
+<SPConfig xmlns="urn:mace:shibboleth:3.0:native:sp:config"
+          xmlns:md="urn:oasis:names:tc:SAML:2.0:metadata"
+          clockSkew="180">
+
+    <OutOfProcess tranLogFormat="%u|%s|%IDP|%i|%ac|%t|%attr|%n|%b|%E|%S|%SS|%L|%UA|%a"/>
+
+
+    <!-- The ApplicationDefaults element is where most of Shibboleth's SAML bits are defined. -->
+    <ApplicationDefaults id="default"
+                         policyId="default"
+                         entityID="https://edview.unilu.ch/shibboleth"
+                         homeURL="https://edview.unilu.ch/"
+                         REMOTE_USER="uniqueID"
+                         cipherSuites="DEFAULT:!EXP:!LOW:!aNULL:!eNULL:!DES:!IDEA:!SEED:!RC4:!3DES:!kRSA:!SSLv2:!SSLv3:!TLSv1:!TLSv1.1"
+                         signing="false"
+                         encryption="false"
+                         attributePrefix="AJP_">
+
+        <!--
+        Controls session lifetimes, address checks, cookie handling, and the protocol handlers.
+        Each Application has an effectively unique handlerURL, which defaults to "/Shibboleth.sso"
+        and should be a relative path, with the SP computing the full value based on the virtual
+        host. Using handlerSSL="true" will force the protocol to be https. You should also set
+        cookieProps to "https" for SSL-only sites. Note that while we default checkAddress to
+        "false", this makes an assertion stolen in transit easier for attackers to misuse.
+        -->
+        <Sessions lifetime="28800"
+                  timeout="3600"
+                  relayState="ss:mem"
+                  checkAddress="false"
+                  consistentAddress="false"
+                  handlerSSL="true"
+                  redirectLimit="host"
+                  sameSiteSession="Lax"
+                  cookieProps="; https; path=/; secure; HttpOnly">
```

Comment:
is httpOnly required here?

Reply:
https://owasp.org/www-community/HttpOnly

Wouldn't HttpOnly be safer?